lexicon/community/bookmark.json

```diff
@@ -0,0 +1,24 @@
+{
+    "lexicon": 1,
+    "id": "community.lexicon.bookmark",
+    "defs": {
+      "main": {
+        "type": "record",
+        "description": "Record bookmarking a link to come back to later.",
+        "key": "tid",
+        "record": {
+          "type": "object",
+          "required": ["subject", "createdAt"],
+          "properties": {
+            "subject": { "type": "string", "format": "uri" },
+            "createdAt": { "type": "string", "format": "datetime" },
+            "tags": {
+                "type": "array",
+                "description": "Tags for content the bookmark may be related to, for example 'news' or 'funny videos'",
+                "items": { "type": "string" }
+            }
```

Comment:
@ngerakines  what do you think of adding tags to bookmarks. Maybe it needs to be called `collections`? The idea is that you can organize the bookmarks into groups for easy viewing related bookmarks

Reply:
I think it's great.